packages/features/portfolio/src/portfolio-feature-shell.tsx

```diff
@@ -45,12 +45,10 @@ export const PortfolioFeatureShell = observer(() => {
   const network = useSelectedNetwork();
   const isAccountLoading = useAccountLoading();
   const isAcctAndBalanceReady = useAccountAndBalanceReady(network);
-  const { hasNoteOrSNote } = usePortfolioNOTETable();
```

Comment:
Are we supposed to remove this?

Reply:
I'll add it back. It doesn't hurt anything that I can see. It just seemed unnecessary and changed nothing when I removed it but maybe there is a edge case I am missing.